modules/benchmark/DESCRIPTION

```diff
@@ -35,6 +35,7 @@ Imports:
 Suggests:
     PEcAn.data.land,
     testthat (>= 2.0.0)
+    bd
```

Comment:
`bd` was the variable used for the name of the BrownDog package. This should be the actual name of the package:

```suggestion
    BrownDog
```

Reply:
@ashiklom sorry for the above mistake. Had a good read about the significance of suggests section in DESCRIPTION.